classes/components/forms/FieldPreparedContent.inc.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * @file classes/components/form/FieldPreparedContent.inc.php
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2000-2021 John Willinsky
```

Comment:
Not sure what the general practice is for updating these/modifying them in new files, but thought I'd flag them just in case.

Reply:
Good spot. I've updated that one but I'll leave the other 2,000 files. :smile: